src/components/ListDetails/ListSummary.vue

```diff
@@ -1,13 +1,43 @@
 <template>
   <div class="list-summary">
-    <h2>List Name</h2>
-    <h2>CountTaskOfList</h2>
+    <h2 class="list-name">{{ listName }}</h2>
+    <h2>{{ countAllTasks }}</h2>
   </div>
 </template>
 
+<script>
+export default {
+  name: "ListSummary",
+  props: {
+    listName: {
+      type: String,
+    },
+    countAllTasks: {
+      type: Number,
+    },
+  },
+};
+</script>
+
 <style scoped>
 .list-summary {
   display: flex;
   justify-content: space-between;
+  height: 5%;
```

Comment:
Hard-coding the parent's height can cause the children's UI to be cut-off, like this:
<img width="958" alt="image" src="https://user-images.githubusercontent.com/25959863/125429107-2157e630-5ad1-4c9a-bd3f-1cdb308cfbcc.png">

Reply:
Updated!